src/component/form.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from "react";
+import {User} from "../type/user"
+
+interface FormProps {
+  onSubmit: (user: User) => void;
+  errorMessage: string;
+  isSubmitting: boolean;
+  buttonText: string;
+}
+
+const Form: React.FC<FormProps> = ({ onSubmit, errorMessage, isSubmitting, buttonText }) => {
```

Comment:
Rename it to user form

Reply:
`const Form = ({ onSubmit, errorMessage, isSubmitting, buttonText }: FormProps) => {`